modules/update-cvo-conditions.adoc

```diff
@@ -0,0 +1,65 @@
+// Module included in the following assemblies:
+//
+// * updating/understanding-update-service.adoc
+
+[id="update-cvo-conditions_{context}"]
+= Understanding CVO conditions
+
+The `ClusterVersion` object sets the conditions that describe the state of the cluster-version operator (CVO).
+
+Failing:: When `Failing` is set to True, the CVO fails to reconcile the cluster with the desired release image. In all cases, the impact on the cluster is that the dependent nodes in the manifest graph may not be reconciled. The graph may be flattened, in which case there are no dependent nodes.
++
+Most reconciliation errors result in `Failing=True`, even though `ClusterOperatorNotAvaliable` has special handling.
+
+NoDesiredImage:: The CVO has not been given a release image to reconcile. When NoDesiredImage is True, it is a CVO coding error. The `desiredUpdate` field returns you to the current CVO's release image.
```

Comment:
"The `desiredUpdate` field returns..." seems to be missing a ["clearing"][1].  I dunno if the web UI has a flow for that, although I'd expect them to.  On the CLI, it's `oc adm upgrade --clear`.

[1]: https://github.com/openshift/cluster-version-operator/blob/c07975c452303469c378bc5b0f70d8763eaca1f3/docs/user/status.md#nodesiredimage

Reply:
Closing this since content has drastically changed.